src/execute/executeService.ts

```diff
@@ -56,6 +56,7 @@ export class ExecuteService {
         }
       }
     }
+    throw new Error(nls.localize('authForAnonymousApexFailed'));
```

Comment:
can you explain this change please?

Reply:
Understood. The error indicates an auth error though? Will that be the case always?